python/lsst/ts/IntegrationTests/configs/auxtel_night_operations_configs.py

```diff
@@ -0,0 +1,118 @@
+# This file is part of ts_IntegrationTests.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import yaml
+
+from .config_registry import registry
+
+
+# Add the AuxTel Night Operations
+# script configurations to the registry.
+
+# auxtel_latiss_cwfs_align
+registry["auxtel_cwfs_align"] = yaml.safe_dump(
+    {
+        "track_target": "",
```

Comment:
I think this might be handled by doing:  `"track_target": {"target_name": "HD164461"}`, but you'd have to check.

Reply:
This does seem to work.